modules/preview-react/side-panel/stories/examples/Basic.tsx

```diff
@@ -20,26 +20,28 @@ export const Basic = () => {
     expanded ? 'expanded' : 'collapsed'
   );
 
+  const expandedContent = (
+    <Flex alignItems="center" paddingY="s" paddingX="s">
+      <Flex marginInlineEnd="s">
+        <AccentIcon icon={rocketIcon} />
+      </Flex>
+      <Text as="h3" typeLevel="body.large" color="licorice500" fontWeight="bold" {...labelProps}>
+        Tasks Panel
+      </Text>
+    </Flex>
+  );
+
   return (
     <CanvasProvider theme={{canvas: {direction}}}>
       <Flex height={320}>
         <SidePanel {...panelProps} onStateTransition={setPanelState}>
           <SidePanel.ToggleButton {...controlProps} />
-          {panelState === 'expanded' && (
-            <Flex alignItems="center" paddingY="s" paddingX="s">
-              <Flex marginInlineEnd="s">
-                <AccentIcon icon={rocketIcon} />
-              </Flex>
-              <Text
-                as="h3"
-                typeLevel="body.large"
-                color="licorice500"
-                fontWeight="bold"
-                {...labelProps}
-              >
-                Tasks Panel
-              </Text>
-            </Flex>
+          {panelState === 'expanded' ? (
+            expandedContent
+          ) : (
+            <span hidden {...labelProps}>
+              Tasks Panel
+            </span>
```

Comment:
Can we use the `Text` component here just to clean this up a bit to look cleaner?

Same would go for the other example as well. 

```suggestion
            <Text as="span" hidden {...labelProps}>
              Tasks Panel
            </Text>
```